fp-bindgen/src/generators/rust_plugin/mod.rs

```diff
@@ -578,3 +592,11 @@ where
 {
     fs::write(file_path, &contents).expect("Could not write bindings file");
 }
+
+fn maybe_quote_str(input: &str) -> String {
+    if input.trim_start().starts_with('{') {
+        input.to_string()
+    } else {
+        format!("\"{input}\"")
```

Comment:
I think we may need to do some escaping of the input, just in case.

Reply:
I think that's the perfect usecase for `Debug` 
```suggestion
        format!("{:?}", input.to_string())
```

Reply:
I replaced the whole formatting code with `toml_edit`, so it's always going to be good going forward.